config/samples/logging_v1_clusterlogforwarder.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: logging.openshift.io/v1
```

Comment:
is this the same as we have in  the hack dir?

Reply:
The sample file is the manifest that we can maintain as proper YAML in the first place and get's appended in the CSV's `alm-examples` annotation, for example [here](https://github.com/openshift/cluster-logging-operator/blob/master/manifests/5.2/cluster-logging.v5.2.0.clusterserviceversion.yaml#L31).

To make use of this one needs to switch the cluster-logging-operator repo to generate the CSV and CRD into the `bundle` dir. We do this in EO for example.

Reply:
It looks then based on @periklis comment these files need to be filled out per something we have in `hack`.

Reply:
Actually we need to extract the samples from the manually crafted `alm-examples` annotations we have in the manifest dir:
https://github.com/openshift/cluster-logging-operator/blob/master/manifests/5.2/cluster-logging.v5.2.0.clusterserviceversion.yaml#L31